src/app/preference/preference-modal/preference-modal.component.html

```diff
@@ -0,0 +1,77 @@
+<ion-content color="light" class="pop-up-content test" text-center>
+  <ng-container *ngIf="!showUpdateModal; else showUpdateModalTemplate">
+    <ion-toolbar class="pn-title-toolbar">
```

Comment:
@ronakamani i dont recommend to copy over the entire [preference.component.ts](https://github.com/intersective/app-v2/blob/AV2-846/Notification-preference-desktop/src/app/preference/preference.component.html) into a different file. it reduces code quality and maintainability.

how about try learn to use [Template syntax](https://angular.io/guide/architecture-components#template-syntax) to insert the instance of component view. 

In your case, 
eg. 
```html
<app-preference-component></app-preference-component>
```